### PR TITLE
ISP response request

### PR DESCRIPTION
**Pull request by jkelvie**
Created helper method for creating Connections Response and ISP Connection Responses in particular

### CODE REVIEW

**Comment by jkelvie** [comment]
First cut of PR based on your work @rmtuckerphx - pretty close to what you had but I left out the enums. They are helpful, but they don't fit cleanly with what we have done to date (we need some sort of Constants class perhaps).

Take a look when you have a chance.

File: src/core/SkillRequest.ts [hunk 1]
```diff
@@ -87,6 +109,23 @@ export class SkillRequest {
         this._json.request.token = token;
         return this;
     }
+
+    public inSkillPurchaseResponse(requestName: string, 
+        productId: string,
```
Comment (rmtuckerphx) [inline src/core/SkillRequest.ts]: The implementation doesn't match the documentation. I prefer the productId as the 3rd param as shown in the docs.

File: test/VirtualAlexaTest.ts [hunk 2]
```diff
@@ -900,3 +900,23 @@ describe("Catalog tests", () => {
         assert.equal(response.slot.value, "cucumber");
     });
 });
+
+describe.only("Connection Response tests", () => {
```
Comment (jperata) [inline test/VirtualAlexaTest.ts]: remove only

File: src/core/SkillRequest.ts (hunk 1 above)
Comment (jkelvie) [reply src/core/SkillRequest.ts]: Hi Mark, which documentation are you referring to? And FYI, I added this based on your code and this blog post:
https://developer.amazon.com/blogs/alexa/post/53c2319a-4cc9-4067-8274-1472a0463d6e/understanding-the-in-skill-purchasing-experience

Comment (rmtuckerphx) [reply src/core/SkillRequest.ts]: You own README.md changed in the PR has the productId as the 3rd param. I agree that it should be 3rd. The code in the PR has it as 2nd param.

**Comment by jkelvie** [comment]
@jperata This is ready to go

**Pull request merged**